packages/loader/container-loader/closeAndGetPendingLocalState.md

```diff
@@ -0,0 +1,31 @@
+# Stashed Changes
+
+When a user goes offline, they are still able to make changes to the document. When the user goes back online, the DDSs will be asked to resubmit the ops, which will use the merging strategy of the DDS to modify the ops on top of new ops so they can be resubmitted and understood by other clients. If `Container.close()` is called while still offline, the changes will be lost. If closed while online, but the changes have not been sequenced yet (i.e. the container is "dirty"), the changes may or may not be lost.
```

Comment:
I wonder if approaching this from another angle could be easier to understand. The gist of the problem is that closing the container while it's in a dirty state may result in data loss, right? Explaining things from that angle (and that while offline, the container definitely cannot transition out of a dirty state) seems to rely on less of the implementation details while still being an accurate enough model.